### ISSUE TITLE
[python] Spinner during script execution

### ISSUE THREAD

**Issue by ImpotStite** [UpsilonNumworks/Upsilon#284]
Bonjour, ça serait bien un logo de chargement qu’un script est en cours à l’a epsilon.
Si vous voyez pas ce dont je veux parler.
C’est le infini à côté du logo batterie quand tu lance un script python.

**Comment by Yaya-Cout** [comment]
I think that the problem of this idea, is that it will slow down script execution, but it's doable. By the way, can use English when writing issues ? If you want to add a French translation, you can do it by adding a separator using `---` with newlines around it.

**Comment by Overengined** [comment]
@Yaya-Cout , I'm not sure that if we just use a static image it would be that bad. What's your opinion ?

**Comment by Yaya-Cout** [comment]
If it is a static image, it would work correctly, but I was talking about an animated spinner.

**Comment by Overengined** [comment]
Yeah, I think think static image is better. Spinner is definitely a bad idea.
